diff --git a/include/nonstd/optional.hpp b/include/nonstd/optional.hpp
index 86e3b40..5cac512 100644
--- a/include/nonstd/optional.hpp
+++ b/include/nonstd/optional.hpp
@@ -56,7 +56,7 @@
 # if defined(_MSC_VER)
 # include <cstddef>     // for _HAS_EXCEPTIONS
 # endif
-# if defined(__cpp_exceptions) || defined(__EXCEPTIONS) || (_HAS_EXCEPTIONS)
+# if defined(__cpp_exceptions) || defined(__EXCEPTIONS) || (defined(_HAS_EXCEPTIONS) && (_HAS_EXCEPTIONS))
 #  define optional_CONFIG_NO_EXCEPTIONS  0
 # else
 #  define optional_CONFIG_NO_EXCEPTIONS  1
